hyperwallet/tests/test_api.py

```diff
@@ -2189,6 +2192,80 @@ def test_list_webhooks_success(self, mock_get):
 
         self.assertTrue(response[0].token, self.data.get('token'))
 
+    '''
+
+    Upload Documents
+
+    '''
+    def test_uploadDocumentsForUser_fail_need_user_token(self):
```

Comment:
It looks like that you added this method. Pls check. this doesn't look related to transfer refund.